components/post_view/post/post.jsx

```diff
@@ -48,11 +48,6 @@ export default class Post extends React.PureComponent {
          */
         consecutivePostByUser: PropTypes.bool,
 
-        /**
-         * Set if the previous post is a comment
-         */
-        previousPostIsComment: PropTypes.bool,
```

Comment:
Here `previousPostIsComment` is in use as part (at least) of the `hasSameRoot` function